server/gateway/docker-compose.server.yml

```diff
@@ -0,0 +1,16 @@
+version: '3'
```

Comment:
what is this file used for and why is it different from r11s?

Reply:
This file runs in conjuction with docker-compose from the /server/ directory.
The r11s file runs in conjunction with docker-compose from the server/r11/ directory

Reply:
Gateway does not build with r11s. This docker-compose is just for Gateway and it expects r11s to also run so that it can talk to it (using the networks part)

Reply:
Yep what I meant above is that this is the docker-compose file that gets run in conjunction with the network built from docker-compose in server directory.
The r11s docker-compose file needs to be run in conjunction with network built from docker-compose in r11s directory